src/lib/dnssd/Types.h

```diff
@@ -194,11 +194,15 @@ struct CommonResolutionData
 };
 
 /// Data that is specific to Operational Discovery of nodes
-struct OperationalNodeData
+struct OperationalNodeData : public CommonResolutionData
```

Comment:
Hmm, so now `ResolvedNodeData` will basically have two copies of `CommonResolutionData`?  One direct, one via `OperationalNodeData`?

That's really confusing as an API consumer.  Do those copies have to be in sync?  If not, which one is the one to use?  If yes, why do we have both?

Seems like there's another intermediate change needed here which is to make this inheritance change and change `ResolvedNodeData` to either just have an `OperationalNodeData` member, or inherit from `OperationalNodeData` and have no data members, or just be an alias for `OperationalNodeData` (and figure out the logging situation).

Reply:
I was actually wondering if we need to provide resolution data in delegate for Operational Discovery. From my understanding, the idea of Operational Browse is to detect new nodes in fabric, application follow up with a resolution request if resolution data is needed for any particular discovered node. Any thoughts on this?

Reply:
As agreed upon, removed inheritance of `CommonResolutionData` from `OperationalNodeData`  as it is not a necessity for Operational Browse.